ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/events/TransactionsReader.scala

```diff
@@ -126,4 +130,32 @@ private[dao] final class TransactionsReader(
       .map(EventsTable.Entry.toGetTransactionResponse)(executionContext)
   }
 
+  def getActiveContracts(
+      activeAt: Offset,
+      filter: FilterRelation,
+      verbose: Boolean,
+  ): Source[(Offset, GetActiveContractsResponse), NotUsed] = {
+    val events =
+      PaginatingAsyncStream(pageSize) { offset =>
+        val query =
+          EventsTable
+            .preparePagedGetActiveContracts(
+              activeAt: Offset,
```

Comment:
Is the type ascription needed here or did you mean to put `activeAt = activeAt`?

Reply:
I meant what you said. I noticed this in the upcoming integration PR, it's fixed there, thanks!